src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```diff
@@ -1569,7 +1569,7 @@ int IAccessible.get_accHelpTopic(out string? pszHelpFile, object childID)
         int count = SystemIAccessible.TryGetChildCount();
 
         // Unclear why this returns null for no children.
-        return count == 0 ? null : count;
+        return (count == 0 || childID is not int id) ? null : SystemIAccessible.TryGetRole(id);
```

Comment:
```suggestion
        return count == 0 ? null : SystemIAccessible.TryGetRole(ChildIdToVARIANT(childID));
```
This seem to be able to cover non-int childID, although I'm not sure in which case it wouldn't be an int. And it's also more consistent with other similar uses of SystemIAccessible.TryGet* when childID is object.